config/locales/de/views/collections.yml

```diff
@@ -29,6 +30,16 @@ de:
         deletion_warning: Sie sind der/die letzte Benutzer:in in dieser Aufgabensammlung. Wenn Sie gehen, wird diese Aufgabensammlung dauerhaft gelöscht! Sind Sie sicher?
       locked_exercise_hint: 'Hinweis: Für Aufgaben mit einem Schloss-Symbol müssen Sie zuerst Zugriff beantragen, um sie anzeigen zu können.'
       noExercisesAdded: Keine Aufgabe hinzugefügt
+      no_other_user: Kein anderer Benutzer hat Zugriff auf diese Aufgabensammlung
+      num_of_other_users:
+        one: Ein anderer Benutzer hat Zugriff auf diese Aufgabensammlung
+        other: "%{count} andere Benutzer haben Zugriff auf diese Aufgabensammlung"
```

Comment:
In the German translation, we are using "gendergerechte Sprache" (with `:`), since CodeHarbor is mainly targeted at schools and universities. Please double check those occurrences, i.e., "Benutzer". There are other examples in this file to look at.